src/web/mjs/connectors/MiniTwoScan.mjs

```diff
@@ -0,0 +1,12 @@
+import WordPressMadara from './templates/WordPressMadara.mjs';
+
+export default class minitwoscan extends WordPressMadara {
```

Comment:
Use pascal casing for class name (same as filename)

Reply:
Okay, updated.